src/definition/accessors/IModify.ts

```diff
@@ -185,6 +185,15 @@ export interface IRoomExtender {
      */
     addMember(user: IUser): IRoomExtender;
 
+    /**
+     * Get a list of users being added to the room.
+     */
+    getMembersBeingAdded(): Array<IUser>;
+
+    /**
+     * Get a list of usernames of users being added to the room.
+     */
+    getUsernamesOfMembersBeingAdded(): Array<string>;
```

Comment:
```suggestion
    getUsernamesOfMembersBeingAdded(): Array<string>;

```

Reply:
@graywolf336 what did you change? Can't see the diff

Reply:
@graywolf336 sorry, I see now, you added a new line 🤦‍♂️